src/app/components/sprint-detail/sprint-detail.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import {Router, ActivatedRoute} from "@angular/router";
+
+@Component({
+  selector: 'scrumly-sprint-detail',
+  templateUrl: './sprint-detail.component.html',
+  styleUrls: ['./sprint-detail.component.scss']
+})
+export class SprintDetailComponent implements OnInit {
+
+  constructor(private _route: ActivatedRoute,
+              private _router: Router) { }
+
+  ngOnInit() {
+    //todo можливо переробити через snapshot якщо є впевненість що маршрут не буде використано повторно
```

Comment:
you should write todo in this style:

TODO(Solomko2): It's possible to write snapshot.....

Please fix this here and in all other files of the commit.